### DISCUSSION TITLE
[Webinar] Wtorki z Security - Azure Active Directory B2C

### DISCUSSION THREAD

**Discussion by wi5nia** [AzureCommunityPL/group#19] [Community Content]
## **Hej,**

razem z @k3aczyk w najblizszy wtorek poprowadzimy sesje o **Azure Active Directory B2C**, na która zapraszamy! :) 
**Poziom sesji**: 100-200    
    
---
[![image](https://user-images.githubusercontent.com/12593852/90639123-8d95d000-e22e-11ea-8138-383ba45729ed.png)](https://info.microsoft.com/CE-SCRTY-WBNR-FY21-08Aug-25-wtorkizsecurityBezpieczenstwokontodc3-SRDEM39088_LP01Registration-ForminBody.html)

**Comment by wi5nia** [comment]
Dzieki za uczestnictwo. 
[Slidedeck](https://1drv.ms/b/s!AmtpV9_qhfAXyJJTolKt9ZBMdKGjCw?e=LId1sP), ktory byl uzyty podczas prezentacji.

**Reply by k3aczyk** [reply]
Dzięki @wi5nia za fajną rozmowę i slidedeck po sesji :)
